mylist.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'myenumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+    print @list
+  end
+end
```

Comment:
- [ ] according to the requirements MyList class should implement `#each` method that yields all members of `@list` in succession.

Reply:
Thank you. We revised the methods & each to class.